tpot/operator_utils.py

```diff
@@ -194,11 +194,12 @@ def TPOTOperatorClassFactory(opsourse, opdict, BaseClass=Operator, ArgBaseClass=
         elif is_regressor(op_obj):
             class_profile['root'] = True
             optype = "Regressor"
-        elif _is_selector(op_obj):
-            optype = "Selector"
         elif _is_transformer(op_obj):
             optype = "Transformer"
-
+        elif _is_selector(op_obj):
```

Comment:
　`_is_selector` need be moved above `_is_transformer` because "Selector" actually is a special kind of "Transformer". If the logic checks "Transformer" first, then there is no Selector in default configuration. It will fix that 2 failed unit tests.